i18n/ru/docusaurus-plugin-content-docs/current/basic/battery_station.mdx

```diff
@@ -7,18 +7,9 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
 <!-- prettier-ignore -->
-:::important Zeitgeist Battery Station is a test network in beta phase.
-You can follow this guide to connect to it but some things might not work yet.
-If you would like to report a bug please open an issue on our GitHub repository.
-Also note that the chain is liable to be reset **at any time without any
-warning.**
-:::
-
-If you want to support and strengthen Battery Station by running your own
-network node, head over to the section
-[Get Connected](battery-station#get-connected). If you want to interact with
-Battery Station, head over to the section
-[Accessing the User Interface](battery-station#accessing-the-user-interface)
+:::important Zeitgeist Battery Station is a test network in beta phase. You can follow this guide to connect to it but some things might not work yet. If you would like to report a bug please open an issue on our GitHub repository. Also note that the chain is liable to be reset **at any time without any warning.** :::
+
+If you want to support and strengthen Battery Station by running your own network node, head over to the section [Get Connected](battery-station#get-connected). If you want to interact with Battery Station, head over to the section [Accessing the User Interface](battery-station#accessing-the-user-interface)
```

Comment:
This re-format breaks the page. It's supposed to look like this: 
<img width="1920" alt="Screenshot 2022-08-09 at 13 32 25" src="https://user-images.githubusercontent.com/49552287/183637328-c04a6535-d95d-4739-ba05-efafa2dc22e1.png">

But it looks like this:
<img width="1920" alt="Screenshot 2022-08-09 at 13 32 12" src="https://user-images.githubusercontent.com/49552287/183637427-d3990f3b-08b8-43aa-94ac-dcd31923d405.png">

Reply:
Are you on the russion translation? Can't tell from your screenshot.

Reply:
I know why, because Crowdin changed the format of this document. I think we can modify this format when translating. What do you think of this?

Reply:
Is you screenshot showing the russian translation or the english version?

> I know why, because Crowdin changed the format of this document. I think we can modify this format when translating. What do you think of this?

I think Crowdin should not change the formatting of the files _at all_. Otherwise, we will keep running into these problems. Can you completely disable the reformatting?

Reply:
ok, will  talk with them about this